tests/v2/test_waku_rln_relay.nim

```diff
@@ -1,560 +1,559 @@
-# 		require(batchSize == receivers.length, "RLN, withdrawBatch: batch size mismatch receivers");
-# 		for (uint256 i = 0; i < batchSize; i++) {
-# 			_withdraw(secrets[i], pubkeyIndexes[i], receivers[i]);
-# 		}
-# 	}
-
-# 	function withdraw(
-# 		uint256 secret,
-# 		uint256 _pubkeyIndex,
-# 		address payable receiver
-# 	) external {
-# 		_withdraw(secret, _pubkeyIndex, receiver);
-# 	}
-
-contract(MembershipContract):
-  proc register(pubkey: Uint256) # external payable
-  # proc registerBatch(pubkeys: seq[Uint256]) # external payable
-  # TODO will add withdraw function after integrating the keyGeneration function (required to compute public keys from secret keys)
-  # proc withdraw(secret: Uint256, pubkeyIndex: Uint256, receiver: Address)
-  # proc withdrawBatch( secrets: seq[Uint256], pubkeyIndex: seq[Uint256], receiver: seq[Address])
-
-proc uploadContract(ethClientAddress: string): Future[Address] {.async.} =
-  let web3 = await newWeb3(ethClientAddress)
-  debug "web3 connected to", ethClientAddress
-
-  # fetch the list of registered accounts
-  let accounts = await web3.provider.eth_accounts()
-  web3.defaultAccount = accounts[1]
-  let add =web3.defaultAccount 
-  debug "contract deployer account address ", add
-
-  var balance = await web3.provider.eth_getBalance(web3.defaultAccount , "latest")
-  debug "Initial account balance: ", balance
-
-  # deploy the poseidon hash first
-  let 
-    hasherReceipt = await web3.deployContract(poseidonHasherCode)
-    hasherAddress = hasherReceipt.contractAddress.get
-  debug "hasher address: ", hasherAddress
-  
-
-  # encode membership contract inputs to 32 bytes zero-padded
-  let 
-    membershipFeeEncoded = encode(MembershipFee).data 
-    depthEncoded = encode(Depth).data 
-    hasherAddressEncoded = encode(hasherAddress).data
-    # this is the contract constructor input
-    contractInput = membershipFeeEncoded & depthEncoded & hasherAddressEncoded
-
-
-  debug "encoded membership fee: ", membershipFeeEncoded
-  debug "encoded depth: ", depthEncoded
-  debug "encoded hasher address: ", hasherAddressEncoded
-  debug "encoded contract input:" , contractInput
-
-  # deploy membership contract with its constructor inputs
-  let receipt = await web3.deployContract(membershipContractCode, contractInput = contractInput)
-  var contractAddress = receipt.contractAddress.get
-  debug "Address of the deployed membership contract: ", contractAddress
-
-  # balance = await web3.provider.eth_getBalance(web3.defaultAccount , "latest")
-  # debug "Account balance after the contract deployment: ", balance
-
-  await web3.close()
-  debug "disconnected from ", ethClientAddress
-
-  return contractAddress
-
-procSuite "Waku rln relay":
-  asyncTest  "contract membership":
-    let contractAddress = await uploadContract(EthClient)
-    # connect to the eth client
-    let web3 = await newWeb3(EthClient)
-    debug "web3 connected to", EthClient
+when defined(rln):
```

Comment:
Why not just disable this whole suite here: https://github.com/status-im/nim-waku/blob/master/tests/all_tests_v2.nim#L15 ?

Reply:
Sure, done!